### PR TITLE
add SwiftLayoutViewConfiguration

### PR DESCRIPTION
**Pull request by oozoofrog**
```swift
deactivable = root {
    UILabel().config { view in
        view.text = "blue"
        view.accessibilityIdentifier = "blue"
        return view
    }
}.active()
```

### CODE REVIEW

File: Sources/SwiftLayout/Extension/File.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//
+//  UIView+Config.swift
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Extension/File.swift]: The current file name is File.swift

File: Tests/SwiftLayoutTests/LayoutDSLTest.swift [hunk 2]
```diff
@@ -77,6 +77,19 @@ extension LayoutDSLTest {
         XCTAssertEqual(red.superview, root)
     }
     
+    func testSimpleWithViewConfiguration() {
+        deactivable = root {
+            UILabel().config { view in
+                view.text = "RED"
+                view.accessibilityIdentifier = "red"
+                return view
```
Comment (gmlwhdtjd) [inline Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: Do I have to return it?
Is there any way to omit return?

File: Tests/SwiftLayoutTests/LayoutDSLTest.swift [hunk 3]
```diff
@@ -78,15 +78,15 @@ extension LayoutDSLTest {
     }
     
     func testSimpleWithViewConfiguration() {
-        deactivable = root {
+        root {
             UILabel().config { view in
                 view.text = "RED"
                 view.accessibilityIdentifier = "red"
                 return view
```
Comment (gmlwhdtjd) [inline Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: I've been trying a few things, and it's practically impossible. What do you think?

File: Tests/SwiftLayoutTests/LayoutDSLTest.swift (hunk 2 above)
Comment (gmlwhdtjd) [reply Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: I've been trying a few things, and it's practically impossible. What do you think?

Comment (oozoofrog) [reply Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: @gmlwhdtjd baton passed

Comment (oozoofrog) [reply Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: > I've been trying a few things, and it's practically impossible. What do you think?

inout?

Comment (gmlwhdtjd) [reply Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: > I've been trying a few things, and it's practically impossible. What do you think?

I found something!

Comment (oozoofrog) [reply Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: > > I've been trying a few things, and it's practically impossible. What do you think?
> 
> inout?

ok ok this should be already tested by you

Comment (oozoofrog) [reply Tests/SwiftLayoutTests/LayoutDSLTest.swift:85]: > > I've been trying a few things, and it's practically impossible. What do you think?
> 
> I found something!

oh!

File: Sources/SwiftLayout/Extension/UIVIew+Layout.swift [hunk 4]
```diff
@@ -21,6 +21,11 @@ public extension Layout where Self: UIView {
         ViewLayout(self, sublayout: build())
     }
     
+    func config(_ config: (Self) -> Void) -> some Layout {
+        config(self)
+        return ViewLayout(self, sublayout: EmptyLayout())
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Extension/UIVIew+Layout.swift:26]: hahahaha this is so what else. I was dumb haha.

Comment (oozoofrog) [reply Sources/SwiftLayout/Extension/UIVIew+Layout.swift:26]: > hahahaha this is so what else. I was dumb haha.

and its more SwiftLayouty. previous protocol just another utility and didn't have any relation on library.

Comment (gmlwhdtjd) [reply Sources/SwiftLayout/Extension/UIVIew+Layout.swift:26]: > and its more SwiftLayouty. previous protocol just another utility and didn't have any relation on library.

yes! I modified the code at the point you said

**Pull request merged**